src/typedb_jupyter/response.py

```diff
@@ -70,17 +70,17 @@ def _group_key(concept):
     @staticmethod
     def _format_json(answer, answer_type):
         if answer_type is ConceptMap:
-            return [concept_map.to_json() for concept_map in answer]
+            return [str(concept_map) for concept_map in answer]
```

Comment:
Idk what this should be

Reply:
This is going to require larger architectural changes. With the removal of JSON format for Get query results and the addition of Fetch queries, it no longer makes sense to specify a response format. Get query results should always use `_format_typeql` and a new implementation will need to be written for `_format_json`, which should always be used by Fetch query results.